packages/vue/src/components/FButton/FButton.cy.ts

```diff
@@ -76,23 +84,42 @@ describe("Primary", () => {
     });
 
     it("text only", () => {
-        cy.mount(createComponent());
+        cy.mount(
+            createComponent({
+                testName: `${Cypress.currentTest.titlePath.join(" ")}`,
```

Comment:
```suggestion
                testName: Cypress.currentTest.titlePath.join(" "),
```

Sen borde vi väl kunna köra det där inne i `createComponent` lika gärna? Så slipper vi parametern och att upprepa detta för varje testfall.